### PR TITLE
Multiple fixes for test cases and serializers

### PR DESCRIPTION
**Pull request by mzbroch**
- renames and fixes nested serializer for BGP Routing Instance model
- adds missing extra_attributes on BGP Routing Instance model
- fixes brief fields on PeerGroupTemplate api representation 
- adds basic BGPRoutingInstanceAPITestCase test cases
- adds basic PeerGroupTemplateAPITestCase test cases
- adds extra_attributes test cases (inheritance too)

### CODE REVIEW

File: nautobot_bgp_models/tests/test_api.py [hunk 1]
```diff
@@ -80,6 +80,93 @@ def test_notes_url_on_object(self):
...
 
 
+class BGPRoutingInstanceAPITestCase(APIViewTestCases.APIViewTestCase):
+    """Test the BGPRoutingInstance API."""
+
+    model = models.BGPRoutingInstance
+    view_namespace = "plugins-api:nautobot_bgp_models"
+    brief_fields = ["display", "id", "url"]
+    bulk_update_data = {
+        "description": "Glenn was here.",
```
Comment (glennmatthews) [inline nautobot_bgp_models/tests/test_api.py:90]: I was? :-O

File: nautobot_bgp_models/tests/test_api.py [hunk 2]
```diff
@@ -80,6 +80,93 @@ def test_notes_url_on_object(self):
...
+            asn=15521, status=status_active, description="Hi ex Premium Internet AS!"
+        )
+
+        cls.create_data = [
+            {
+                "description": "Hello World!",
+                "autonomous_system": asn_8545.pk,
+                "device": device_1.pk,
+                "router_id": address.pk,
+                "extra_attributes": {"key1": 1, "key2": {"nested_key2": "nested_value2", "nk2": 2}},
```
Comment (glennmatthews) [inline nautobot_bgp_models/tests/test_api.py:139]: Do we need any tests above-and-beyond the generic base tests specifically for the `extra_attributes` field?

File: nautobot_bgp_models/tests/test_api.py (hunk 1 above)
Comment (mzbroch) [reply nautobot_bgp_models/tests/test_api.py:90]: Originally:)

File: nautobot_bgp_models/tests/test_api.py (hunk 2 above)
Comment (mzbroch) [reply nautobot_bgp_models/tests/test_api.py:242]: Multiple test cases added for extra_attributes, including inheritance of extra attributes

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**